public/containers/NotificationConfig/NotificationCallout.tsx

```diff
@@ -0,0 +1,75 @@
+import { EuiCallOut, EuiLink, EuiSpacer } from "@elastic/eui";
+import React from "react";
+import { ROUTES } from "../../utils/constants";
+import { ActionType, OperationType, OperationTypeMapTitle } from "../../pages/Notifications/constant";
+import { getPermissionValue } from "./hooks";
+
+export interface INotificationCalloutProps {
+  hasDefaultNotification: boolean;
+  operationType?: OperationType;
+  actionType: ActionType;
+  permissionForViewLRON: boolean;
+  permissionForCreateLRON: boolean;
+}
+
+export default function NotificationCallout(props: INotificationCalloutProps) {
+  const { hasDefaultNotification, operationType, actionType, permissionForCreateLRON, permissionForViewLRON } = props;
+  const permissionValue = getPermissionValue(permissionForViewLRON, permissionForCreateLRON, hasDefaultNotification);
+
+  if (permissionValue === "110") {
+    return null;
+  }
```

Comment:
what does 110 means? better to have enum to define all values.

Reply:
Already add comment on permissionValue.